src/main/java/org/jabref/logic/importer/fileformat/CiteSeerParser.java

```diff
@@ -0,0 +1,54 @@
+package org.jabref.logic.importer.fileformat;
+
+import java.net.CookieHandler;
+import java.net.CookieManager;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jabref.logic.importer.ParseException;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.StandardField;
+
+import kong.unirest.json.JSONArray;
+import kong.unirest.json.JSONObject;
+
+public class CiteSeerParser {
+
+//    private static final Pattern PAPERID = Pattern.compile("pid/[0-9a-zA-Z]+", Pattern.MULTILINE);
```

Comment:
Can be removed?

Reply:
Yes. I had intended to use this pattern for a different approach. Will remove.